main.rb

```diff
@@ -0,0 +1,34 @@
+require_relative './book'
+require_relative './person'
+require_relative './teacher'
+require_relative './student'
+require_relative './classroom'
+require_relative './app'
+
+@books = []
+@students = []
+@teachers = []
+
+def main
+  loop do
+    display_menu
+    choice = gets.chomp.to_i
+    case choice
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      add_person
+    when 4
+      add_book
+    when 5
+      break
+    else
+      puts 'Invalid choice!'
+    end
+  end
+end
+
```

Comment:
While you did a good job working on the project requirements. There are some fields you missed out in your console app Your console app is supposed to a have the following fields for it to  be fully functional.

Required fields:

"1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"

But for you your console app only has the following:

Take a look at the screen shot below:
![Screenshot 2023-05-26 at 12 50 24 PM](https://github.com/fuadmoin/school-library/assets/79506105/7aec3a37-8224-45f2-8bef-46b042cbb2f3)

it would be nice if you added the following:

Create a rental"
List all rentals for a given person id
And they should be fully functional.